test/extensions/tracers/xray/tracer_test.cc

```diff
@@ -355,7 +357,7 @@ TEST_F(XRayTracerTest, UseExistingHeaderInformation) {
   xray_header.trace_id_ = "a";
   xray_header.parent_id_ = "b";
   constexpr auto span_name = "my span";
-  constexpr auto operation_name = "my operation";
+  constexpr auto operation_name = "ingress";
```

Comment:
style-guide nit: use explicit types for these.

Reply:
done, thanks. here and other places as well.